src/reducers/todoReducers.js

```diff
@@ -26,6 +28,17 @@ const todoReducers = (state = initialData, action) => {
         list: newList,
       };
 
+    case actionType.COMPLETE_TODO:
+      const completedTaskIndex = state.list.findIndex(
+        (e) => e.id === action.id
```

Comment:
Could we be more explicit about naming arguments ?
```suggestion
        (task) => task.id === action.id
```

Reply:
changed that. Thanks